multinet/api/tests/test_csv_network.py

```diff
@@ -192,3 +192,65 @@ def test_rest_create_csv_network_already_exists(
     )
     assert r.status_code == 400
     assert r.json() == 'Network already exists'
+
+
+@pytest.mark.django_db
+def test_create_csv_network_missing_joins(workspace, table_factory):
+    edge_table: Table = table_factory(workspace=workspace)
```

Comment:
Should we be using `EdgeTableFactory` for this one? I haven't fully grasped all the behavioral implications of edge vs. non-edge Tables.

Reply:
In this case no, it's a bit of a misnomer. While this table _is_ an edge table in the sense that it contains the edge descriptions of the data, it's actually defined as a node table in our database, as this data is yet to be "linked" to form an actual edge table. 

The csv network builder is the only place this happens afaik, as it involves uploading node tables and then linking them to create an edge table and network from the specification.